chore: rename git_push_stack to mergify_cli (#162)
Mergifyio/mergify-cli: README.md

> # Mergify-cli

Reviewer: ```suggestion
# Mergify CLI
```

branches all commit sha changes and maybe you added a commit, or removed one, or
even reordered them. Making impossible to track what going on with the commit index or sha.

> Mergify-cli uses the same technique as other Code Review tools like [Gerrit](https://www.gerritcodereview.com/)

Reviewer: ```suggestion
Mergify stacked PRs uses the same technique as other Code Review tools like [Gerrit](https://www.gerritcodereview.com/)
```

Like Gerrit, it makes no compromise and it assumes your commit messages are well
written and use them in title and body of pull requests.

> Also Mergify-cli leverages the GitHub API to do the tidy jobs, keeping your

Reviewer: ```suggestion
Also, Mergify CLI leverages the GitHub API to do tidy jobs, keeping your
```

## How the git workflow changes:

> To uses Mergify-cli you have to replace:

Reviewer: ```suggestion
To uses Mergify stacked PRs, you have to replace:
```

```bash
...do some change and commit...

> $ mrgfy push-stack

Reviewer: ```suggestion
$ mergify push-stack
```
I don't think we want to truncate the name here. It's easy to `alias` for anyone if they want too.

Author: I will take this comment in account, the others are not relevant since I totally re-write readme.me in the last commit of the stack. I should have removed it from this commit and that is what I will do.
